.gitignore

```diff
@@ -135,3 +135,4 @@ tmp_cache
 
 # python-language-server / Rope
 .ropeproject
+*.code-workspace
```

Comment:
Seems this line is unrelated to the PR. Please remove it.

```suggestion
```

Reply:
Just added that to ignore my VSCode workspace file.

Reply:
Please follow up the comments

Reply:
Removed it.